### PR TITLE
Fix function-webhook

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Fix `function-webhook`.

Changes proposed in this pull request:

- Add permissions for: `leases.coordination.k8s.io` in `serverless-webhook` cluster role.

**Related issue(s)**

See also: https://github.com/kyma-project/kyma/issues/10599

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -77,8 +77,8 @@ global:
 images:
   manager:
     repository: "eu.gcr.io/kyma-project/function-controller"
-    tag: "8a8658b2"
-    pullPolicy: IfNotPresent
+    tag: "PR-10574"
+    pullPolicy: Always
```
Comment (m00g3n) [inline resources/serverless/values.yaml:81]: remove before merge

File: components/function-controller/go.mod [hunk 2]
```diff
@@ -18,13 +18,12 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/vrischmann/envconfig v1.3.0
 	go.uber.org/zap v1.16.0
-	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de
-	google.golang.org/protobuf v1.25.0
```
Comment (kfurgol) [inline components/function-controller/go.mod:22]: please double check if that import is safe to remove

**Review by kfurgol — APPROVED** [review]
lgtm

**Review by kfurgol — APPROVED** [review]
lgtm

**Pull request merged**